.github/CODEOWNERS

```diff
@@ -0,0 +1,86 @@
+# Lists some code owners.
+#
+# A codeowner just oversees some part of the codebase. If an owned file is changed then the
+# corresponding codeowner receives a review request. An approval of the codeowner might be
+# required for merging a PR (depends on repository settings).
+#
+# For details about syntax, see:
+# https://help.github.com/en/articles/about-code-owners
+# But here are some important notes:
+#
+# - Glob syntax is git-like, e.g. `/core` means the core directory in the root, unlike `core`
+#   which can be everywhere.
+# - Multiple owners are supported.
+# - Either handle (e.g, @github_user or @github/team) or email can be used. Keep in mind,
+#   that handles might work better because they are more recognizable on GitHub,
+#   eyou can use them for mentioning unlike an email.
+# - The latest matching rule, if multiple, takes precedence.
+
+# Top levels
+/docker @paritytech/ci @paritytech/release-engineering
+/prdoc @paritytech/release-engineering
+/docs @paritytech/docs-audit
+/substrate @paritytech/SubstrateTeamLeads
+/polkadot @paritytech/polkadot-review
+/cumulus @paritytech/cumulus-locks-review
+.rustfmt.toml @paritytech/core-devs
+Cargo.toml @paritytech/core-devs
+*.md @paritytech/docs-audit
+
```

Comment:
We will have a `bridges` folder soon, maybe you can already add it here https://github.com/paritytech/polkadot-sdk/pull/1392

Reply:
Sure, happy to do so.

Reply:
I added that in the current PR as @serban300 is ooo atm and I don't want to rebase his PR "in his back" :)